### ISSUE TITLE
Periodically run ANALYZE/VACUUM on Metabase DB

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#1762]
> This command should be run from time to time to improve the query plans generated by the optimizer.

As a scheduled task it probably makes sense to run DB maintenance commands every once in a while to keep things fast and spiffy. 
-  [H2](http://h2database.com/html/grammar.html#analyze)
-  [Postgres](http://www.postgresql.org/docs/current/static/sql-vacuum.html)

**Comment by salsakran** [comment]
@agilliland any opinions on frequency? Think this would be a good no-UX issue.

I'd say once a week seems most sane.

**Comment by salsakran** [comment]
Also, this is imo low priority as our transaction volume on the application database (and especially delete load) is pretty low.

**Comment by agilliland** [comment]
yeah, once a week is plenty.  this really isn't that important though.

**Comment by paoliniluis** [comment]
OPTIMIZE TABLE for MySQL would be nice as well

**Comment by paoliniluis** [comment]
Feeling that we could solve this with some lines of documentation
